functions/package.json

```diff
@@ -15,7 +15,8 @@
     "start": "firebase serve --only functions:assistant",
     "start:alexa": "bst proxy lambda index-alexa",
     "deploy": "firebase deploy --only functions:assistant",
-    "test": "npm run lint && npm run mocha"
+    "test": "npm run lint && npm run mocha",
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha -- -- -u exports -R spec tests/**/* && ./node_modules/.bin/codecov"
```

Comment:
shouldn't you use `nyc` instead https://istanbul.js.org/docs/tutorials/mocha/ ?

Reply:
hey @hyzhak using `nyc` has lead me to a weird error. Long story short, `nyc` requires superuser privilages whereas `codecov` shows weird errors when used with sudo. Sometimes it will say supply the token (even when env variable is set), sometimes it says server hang up, sometimes says token is not valid UUID. Without sudo, it works perfectly. Can we just use `istanbul cover` instead of `nyc` ?
moreover, `istanbul cover` automatically writes coverage reports to a folder called `coverage` while `nyc` doesnt create it when its not there. I'm having hard time with this, can you please look into it??

Reply:
moreover, `nyc` is just CLI for `istanbul`, so it should be okay.